diff --git a/node_modules/@favware/cliff-jumper/.bun-tag-a16bd6ab2a1a5720 b/.bun-tag-a16bd6ab2a1a5720
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/node_modules/@favware/cliff-jumper/.bun-tag-b6b8f2b0cac4d163 b/.bun-tag-b6b8f2b0cac4d163
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/commands/install-dependencies.js b/dist/commands/install-dependencies.js
index a32c9d894b28c997741f3ea297eda3a76435374c..4ff3c2f9689255571f31bc41900d6415493cb15d 100644
--- a/dist/commands/install-dependencies.js
+++ b/dist/commands/install-dependencies.js
@@ -20,6 +20,8 @@ function packageManager(packageManagerUsed) {
         case 'yarn-v3':
         case 'yarn-v4':
             return 'yarn';
+        case 'bun':
+            return 'bun';
     }
 }
 //# sourceMappingURL=install-dependencies.js.map
\ No newline at end of file
diff --git a/dist/lib/utils.js b/dist/lib/utils.js
index 58a772fcef25b4b32602e185f2540914e4c77f48..7384c81db0008a7c039275719888f42c5a904e77 100644
--- a/dist/lib/utils.js
+++ b/dist/lib/utils.js
@@ -19,6 +19,8 @@ export function resolveUsedPackageManager() {
         return 'npm';
     if (npmConfigUserAgentEnvVar.startsWith('pnpm/'))
         return 'pnpm';
+    if (npmConfigUserAgentEnvVar.startsWith('bun/'))
+        return 'bun';
     if (npmConfigUserAgentEnvVar.startsWith('yarn/')) {
         if (npmConfigUserAgentEnvVar.startsWith('yarn/1'))
             return 'yarn-v1';
@@ -35,6 +37,8 @@ export function resolvePublishCommand(packageManagerUsed) {
     switch (packageManagerUsed) {
         case 'pnpm':
             return 'pnpm publish';
+        case 'bun':
+            return 'bun publish';
         case 'yarn-v1':
             return 'yarn publish';
         case 'yarn-v2':
@@ -49,6 +53,8 @@ export function resolveInstallCommand(packageManagerUsed) {
     switch (packageManagerUsed) {
         case 'pnpm':
             return 'pnpm install';
+        case 'bun':
+            return 'bun install';
         case 'yarn-v1':
             return 'yarn install';
         case 'yarn-v2':
